app/orchestrators/ReportOrchestrator.scala

```diff
@@ -40,18 +40,17 @@ class ReportOrchestrator @Inject()(reportRepository: ReportRepository,
 
   private def notifyProfessionalOfNewReport(report: Report, company: Company): Future[Report] = {
     companyAccessRepository.fetchAdmins(company).flatMap(admins => {
-      val adminsWithEmail = admins.filter(_.email.isDefined)
-      if (adminsWithEmail.nonEmpty) {
+      if (admins.nonEmpty) {
         mailerService.sendEmail(
           from = mailFrom,
-          recipients = adminsWithEmail.flatMap(_.email): _*)(
+          recipients = admins.map(_.email): _*)(
           subject = "Nouveau signalement",
           bodyHtml = views.html.mails.professional.reportNotification(report).toString,
           attachments = Seq(
             AttachmentFile("logo-signal-conso.png", environment.getFile("/appfiles/logo-signal-conso.png"), contentId = Some("logo"))
           )
         )
-        val user = adminsWithEmail.head     // We must chose one as Event links to a single User
+        val user = admins.head     // We must chose one as Event links to a single User
         eventRepository.createEvent(
```

Comment:
Peut-être qu'il faudra prévoir de modifier le type de la colonne `user_id` de la table `events` en `UUID[]` (mais on perdra le FK) ? ou créer un `event` par `user` ?
Cela permettrait de faciliter la lecture de l'historique par les admins SignalConso en particulier lorsque les Pros vont commencer à supprimer des users

Reply:
Oui on perdrait les FK, non compatibles avec les array (cf https://stackoverflow.com/questions/41054507/postgresql-array-of-elements-that-each-are-a-foreign-key). Mais bon, je ne sais pas si c'est grave dans la mesure où on ne doit pas supprimer de User...

Reply:
D'ailleurs les Pros ne peuvent pas supprimer les User. Ils suppriment juste des accès.